protocol/messenger_backup_handler.go

```diff
@@ -304,22 +304,27 @@ func (m *Messenger) handleWatchOnlyAccount(message *protobuf.SyncAccount) error
 
 func (m *Messenger) handleSyncedCommunities(state *ReceivedMessageState, message *protobuf.Backup) []error {
 	var errors []error
+	handledCommunities := make(map[string]struct{})
```

Comment:
I don't know that code well enough, so I didn't want to introduce any major change.

As I could see, I was calling `request` more than just once. I think it's because when we fetch we get multiple copies of the backup?

I didn't wrap the whole loop in the `alreadyHandled` condition, because if my assumption that we can get multiple copies of the community is right, I don't want to miss  the latest one if we handled an old one.

But maybe that whole code is useless. I don't mind removing it. I just found it weird that I was calling the request multiple times.

Reply:
https://github.com/status-im/status-go/commit/1d6789150414c1344f319213412dfb68ecd5845c

I created a diff to explain what I wanted to say (code says more than thousand words :sweat_smile: )

Reply:
Very clean! I did think of doing that, but I got lazy I guess :sweat_smile: I'll add your commit to my PR